product_multi_price/data/multi_price_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo noupdate="1">
+
+    <record id="price_1" model="product.multi.price.name">
+        <field name="name">price_1</field>
```

Comment:
Why do yo need this hardcoded data?

Reply:
Nothing really. I'll leave it as demo data for easier testing.